docs/swarm/swarm-tutorial/create-swarm.md

```diff
@@ -38,7 +38,10 @@ node. For example, the tutorial uses a machine named `manager1`.
         --token SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
         192.168.99.100:2377
 
-    To add a manager to this swarm, run 'docker swarm join-token manager' and follow the instructions.
```

Comment:
Oh, erm, I now see that changing this is not correct; this output changed slightly, and what's documented here is the correct output for 1.12.1. We changed this, because people sometimes copy/pasted the wrong line, and added nodes as manager, not as a worker, so we now hide the join-token for managers by default, but allow you to get that using `docker swarm join-token manager`